### ISSUE TITLE
`dcv_delegation_records` key does not exist in `CustomHostnameGetResponse.SSL` and `CustomHostnameEditResponse.SSL`

### ISSUE THREAD

**Issue by aperture147** [cloudflare/cloudflare-typescript#2677]
### Confirm this is a Typescript library issue and not an underlying Cloudflare API issue

- [x] This is an issue with the Typescript library

### Describe the bug

At the moment Cloudflare API is responding `dcv_delegation_records` key in the `ssl` object, but the `SSL` object does not have this key. This does not make any error in the compile time or run time, but result an false error report on the TypeScript type check while coding

### To Reproduce

1. Create Cloudflare client: `const cf = Cloudflare({ apiToken: c.env.CLOUDFLARE_API_TOKEN })`
2. Create a new hostname (or edit an existing one): `const cfData  = cf.customHostnames.create({...})`
3. Get the `dcv_delegation_records` key from the response: `cfData.ssl.dcv_delegation_records`, this will result an error: `Property 'dcv_delegation_records' does not exist on type 'SSL'`

### Code snippets

```Typescript

```

### OS

any

### Runtime version

any

### Library version

5.1.0

**Comment by mohankumarelec** [comment]
This issue is still persisting
